script.js

```diff
@@ -0,0 +1,41 @@
+const images = document.querySelectorAll('.slider__line img');
```

Comment:
Можна трохи перетасувати:
```suggestion
const sliderLine = document.querySelector('.slider__line');
const images = sliderLine.querySelectorAll('img');
```
Але було б краще використовувати класи.
```suggestion
const sliderLine = document.querySelector('.slider__line');
const images = sliderLine.querySelectorAll('.slider__img');
```